pact/provider/pact-broker-loader/src/main/java/org/arquillian/algeron/pact/provider/core/loader/pactbroker/PactBrokerLoader.java

```diff
@@ -156,6 +151,79 @@ public void setHttpResponseCallable(Callable<HttpResponse> httpResponseCallable)
         this.httpResponseCallable = httpResponseCallable;
     }
 
+    static class PactBrokerImpl implements PactBroker {
+
+        private static final String HOST = "host";
+        private static final String PORT = "port";
+        private static final String PROTOCOL = "protocol";
+        private static final String TAGS = "tags";
+
+        private String host = "";
+        private String port = "";
+        private String protocol = "http";
+        private List<String> tags = new ArrayList<>();
+
+        public PactBrokerImpl(Map<String, Object> configuration) {
+
+            if (configuration.containsKey(HOST)) {
+                this.host = (String) configuration.get(HOST);
+            } else {
+                throw new IllegalArgumentException("Host field is mandatory in Pact Broker Retriever.");
+            }
+
+            if (configuration.containsKey(PORT)) {
+                this.port = (String) configuration.get(PORT);
+            } else {
+                throw new IllegalArgumentException("Port field is mandatory in Pact Broker Retriever.");
+            }
```

Comment:
Can't we think about some method for these repetitive operations? (one for required - throwing exception if not present and one for optional values like `PROTOCOL` and alike)